client/src/components/files/FileList.tsx

```diff
@@ -2,16 +2,14 @@ import React from 'react';
 import {Table, Button, Icon} from 'semantic-ui-react';
 
 type Props = {
-  title: string;
   extensionFilter?: string;
```

Comment:
If `extensionFilter` is no longer used, this can probably be deleted.

Reply:
Oh yes, missed this somehow. Will do.